packages/app-builder-lib/src/node-module-collector/packageManager.ts

```diff
@@ -70,18 +72,24 @@ export function detectPackageManagerByLockfile(cwd: string): PM | null {
   const yarn = has("yarn.lock")
   const pnpm = has("pnpm-lock.yaml")
   const npm = has("package-lock.json")
+  const bun = has("bun.lock") || has("bun.lockb")
 
   const detected: PM[] = []
   if (yarn) detected.push(PM.YARN)
   if (pnpm) detected.push(PM.PNPM)
   if (npm) detected.push(PM.NPM)
+  if (bun) detected.push(PM.BUN)
```

Comment:
Please consider prettier settings for mult-line `if` statements. This can all be simplified to the below since `has` is already pretty self-explanatory.
```
const detected: PM[] = []
if (has("yarn.lock")) {
  detected.push(PM.YARN)
}
if (has("pnpm-lock.yaml")) {
  detected.push(PM.PNPM)
}
if (has("package-lock.json")) {
  detected.push(PM.NPM)
}
if (has("bun.lock") || has("bun.lockb")) {
  detected.push(PM.BUN)
}
```

Reply:
Fixed